compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -131,15 +169,32 @@ void RecordMinMax::profileData(const std::string &input_data_path)
   }
 
   auto minmax_map = _observer->minMaxData()->getMap();
-  (void *)minmax_map;
+  for (auto iter = minmax_map->begin(); iter != minmax_map->end(); ++iter)
```

Comment:
(trivial)
```suggestion
  for (auto &it : minmax_map)
```

Reply:
@jinevening , do you want to apply this proposal or want to merge as-is?

Reply:
@seanshpark Can you merge this as-is? I will make the change later.